intl/msg_hash_us.h

```diff
@@ -1464,6 +1480,34 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_VIDEO_DRIVER,
    "Video driver to use."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_DRIVER_GL,
+   "OpenGL Video driver. This driver allows libretro GL cores to be used in addition to software-rendered core implementations. Performance for software-rendered and libretro GL core implementations is dependent on your graphics card's underlying GL driver."
+   )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_DRIVER_SDL2,
+   "SDL 2 software-rendered video driver. Performance for software-rendered libretro core implementations is dependent on your platform SDL implementation."
+   )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_DRIVER_SDL1,
+   "SDL 1.2 software-rendered video driver. Performance is considered to be suboptimal. Consider using it only as a last resort."
+   )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_DRIVER_EXYNOS,
+   "A low-level Exynos video driver that uses the G2D block in Samsung Exynos SoC for blit operations. Performance for software rendered cores should be optimal."
+   )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_DRIVER_DRM,
+   "Plain DRM Video Driver. This is a low-level video driver using libdrm for hardware scaling using GPU overlays."
```

Comment:
"Video Driver" uppercasing might be superfluous, considering those words are not uppercased elsewhere.